lib/galaxy/objectstore/__init__.py

```diff
@@ -38,9 +39,10 @@
 log = logging.getLogger(__name__)
 
 
-class ObjectStore(object):
+class IObjectStore(object):
```

Comment:
I'd just call it ObjectStore still - we haven't prefixed other interfaces in Galaxy with I.

Reply:
I refactored this back.